NEWPART.md

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
Should be in `docs/`?

Reply:
Yeah, makes sense.   The question is, /only/ in docs as .rst, or also have an .md version?   I noticed that the getting started readme exists in both formats.

Reply:
I'll ask for re-reviews after updating the content here, and then when it looks good, I'll convert it to an .rst file under docs/.

Reply:
@mkurc-ant , @litghost , the new doc (docs/newpart.rst) is on readthedocs, ready for review.   Ignore the NEWPART.md file; I'll delete it before merging.   Start here:  [https://symbiflow--1486.org.readthedocs.build/projects/prjxray/en/1486/db_dev_process/index.html](https://symbiflow--1486.org.readthedocs.build/projects/prjxray/en/1486/db_dev_process/index.html) (to see where I've put the new page in the hierarchy), then click on "Guide to adding...".

Reply:
Is it time to delete this file?